recipe-client-addon/jar.mn

```diff
@@ -3,8 +3,9 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 [features/shield-recipe-client@mozilla.org] chrome.jar:
-% resource shield-recipe-client %content/
-  content/lib/ (./lib/*)
-  content/data/ (./data/*)
-  content/skin/  (skin/*)
-  content/vendor/ (./vendor/*)
+% resource shield-recipe-client %
```

Comment:
Note that this technically exposes all the resources shipped with the add-on to web content.  We're changing that in https://bugzilla.mozilla.org/show_bug.cgi?id=863246 but it might be something to be more cautious about here? Could use 2 resource contexts (shield-recipe-client-public or something) to mitigate this.